plugins/osdn/ovs/controller.go

```diff
@@ -325,6 +348,125 @@ func (plugin *ovsPlugin) SetupSDN(localSubnetCIDR, clusterNetworkCIDR, servicesN
 	return true, nil
 }
 
+func (plugin *ovsPlugin) SetupEgressFirewall() error {
+	plugin.curFwTable = -1
+	fw, err := plugin.Registry.GetEgressFirewall()
+	if err == nil {
+		plugin.fwRules = fw.Rules
```

Comment:
oops, yeah, there was some rewriting when I added the kerrors.IsNotFound() check. It's actually the other assignment that's wrong, not this one